### ISSUE TITLE
IPv6 stack support

### ISSUE THREAD

**Issue by SoniEx2** [MightyPirates/OpenComputers#2463]
Every computer should have its own IPv6 address. We live in a world of docker, virtualization, UPnP, etc so it makes sense to have a full IPv6 stack. And with IPv6 this is fine because we have more IPv6 IPs than we know what to do with.

Benefits:
- UDP
- Servers
- IoT
- NTP
- Etc

**Comment by magik6k** [comment]
There is IP stack for plan9k(oppm:plan9k-ip) and I have another unpublished IP stack for OpenOS. Both currently only support IPv4/UDP. Adding IPv6 should be trival, TCP would be hard. Public routability can be archieved via external go server.  I don't think this needs be implemented directly by the mod. Other than that you can't implement this without requiring special privileges making it even more out of scope of the mod.

TL;DR can't be easily done in mod, was done in lua.

**Comment by SoniEx2** [comment]
Working on getting userspace IPv6-only switches into linux. Should be possible in the near future.

I have a /64 might aswell use it, right? Each computer with its own public IPv6 address (because IPv6 can do that) and being able to bind to ports < 1024 without root privileges (if you have a whole IP all for yourself you don't need to worry about port conflicts).
